homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -0,0 +1,96 @@
+"""
+Support for INSTEON dimmers via PowerLinc Modem.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/insteon_plm/
+"""
+import logging
+import asyncio
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ['insteon_plm']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up the INSTEON PLM device class for the hass platform."""
+    _LOGGER.info('Provisioning Insteon PLM Binary Sensors')
+
+    plm = hass.data['insteon_plm']
+
+    def async_plm_binarysensor_callback(device):
+        """New device detected from transport."""
+        name = device['address']
+        address = device['address_hex']
+
+        _LOGGER.info('New INSTEON PLM binary sensor device: %s (%s)',
+                     name, address)
+        hass.async_add_job(async_add_devices(
+            [InsteonPLMBinarySensorDevice(hass, plm, address, name)]))
```

Comment:
You not need the `async_add_devices` you can create your entity and call `entity.async_update_ha_state`